library/validation/validation_script.js.php

```diff
@@ -0,0 +1,134 @@
+<?php
+if($GLOBALS['new_validate']) {
+?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/moment.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/validate/validate.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/validate/validate_extend.js"></script>
+<?php
+ }
+?>
+
+<script language='JavaScript'>
+
+    function submitme(new_validate,e,form_id) {
```

Comment:
Don't make this hard on other developers. Add comments to document what the function does. Be sure to note that it calls the existing validate() function.